### ISSUE TITLE
Error when duplicating an image *having >2 dimensions* with a selection outside the image borders

### ISSUE THREAD

**Issue by bvandenbroek** [imagej/ImageJ#258]
Hi,

This issue is very much related to issue #254, It was resolved by Wayne, but apparently only for images with 2 dimensions (x and y).
This problem still remains for all other images, as far I can see: duplicating an image with a selection outside the image borders causes a crash.

![image](https://github.com/user-attachments/assets/4c052fd6-b99a-419c-ab20-118675495f41)

Reproduce with this IJ1 macro:
```
run("Neuron (5 channels)");
makeRectangle(800, 800, 50, 50);
run("Duplicate...", "title=test ignore");
```
or
```
run("Confocal Series");
makeRectangle(800, 800, 50, 50);
run("Duplicate...", "title=test ignore");
```

Best regards,
Bram

**Comment by rasband** [comment]
This bug will be fixed in the ImageJ 1.54m1 daily build, due early next week.

**Comment by rasband** [comment]
This bug is fixed in the ImageJ 1.54m1 daily build. The commit is at https://github.com/imagej/ImageJ/commit/7d8d18b0dd78582cbc7ebf3cde745d5a8ac32007.
